sdks/python/hatchet_sdk/runnables/workflow.py

```diff
@@ -455,6 +466,91 @@ def inner(
 
         return inner
 
+    def durable(
```

Comment:
nit: i think implicit durable_task would make this more clear

Reply:
good call, I'll change that 👍